test/rendering.spec.js

```diff
@@ -54,6 +54,18 @@ describe('Render component (DOM element) inside another', () => {
     expect(renderedComponent).toHaveClass('with-class');
     expect(renderedComponent).toBe(aComponent);
   });
+
+  it('when destroy component , then parent component should be not have rendered component', () => {
+    const renderedComponent = render({
+      component: aComponent,
+      inside: aParent,
+      withClasses: 'with-class',
+    });
+
+    expect(aParent).toContainElement(renderedComponent);
```

Comment:
Nie jestem pewien czy to jest potrzebne. Bo już jest testowane przy render :) 
Powinniśmy się trzymać zasady jedna LOGICZNA asercja na test. Czyli dla render sprawdzamy czy wyrenderowal. A dla render + destory czy go nie ma. Bo juz wiemy z wczesniejszych testow, ze render dziala poprawnie.